### PR TITLE
Merge 18.9 code freeze into develop

### PR DESCRIPTION
**Pull request by jkmassel**
Merges the `18.9` code freeze into `develop`, including:
- Version number bumps
- Release toolkit update
- Fastlane + Fastlane Plugin updates
- Production Pods
- Translation pipeline fixes

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -418,19 +418,12 @@ platform :ios do
     en_lproj = 'WordPress/Resources/en.lproj'
     ios_generate_strings_file_from_code(
       paths: ['WordPress/', 'Pods/WordPress*/', 'Pods/WPMediaPicker/', 'WordPressShared/WordPressShared/', 'Pods/Gutenberg/'],
-      exclude: ['*Vendor*', 'WordPress/WordPressTest/I18n.swift', 'WordPress/WordPressStatsWidgets/Views/Localization/AppLocalizedString.swift'],
+      exclude: ['*Vendor*', 'WordPress/WordPressTest/I18n.swift', 'WordPress/WordPressStatsWidgets/Views/Localization/AppLocalizedString.swift', 'WordPress/Classes/Utility/AppLocalizedString.swift'],
```
Comment (jkmassel) [inline fastlane/Fastfile]: @AliSoftware – I needed to make this change to get this method to work – LMK if that looks correct? I think we could potentially also use a wildcard here, but I kept it simple to be safe.

File: fastlane/Fastfile [hunk 2]
```diff
@@ -418,19 +418,12 @@ platform :ios do
...
       routines: ['AppLocalizedString'],
       output_dir: en_lproj
     )
 
-    # @TODO: Remove once we migrated the codebase of the widgets to use the app bundle for .strings
```
Comment (jkmassel) [inline fastlane/Fastfile:426]: @AliSoftware – I needed to remove these in order to complete the code freeze – I believe these were resolve as part of #17630 and #17636 , but LMK if anything doesn't look right?

File: fastlane/Fastfile (hunk 1 above)
Comment (AliSoftware) [reply fastlane/Fastfile]: The file was actually moved during https://github.com/wordpress-mobile/WordPress-iOS/pull/17630, from its previous location (in `WordPress/WordPressStatsWidgets/Views/Localization/AppLocalizedString.swift`) to the new one you added; so you should just replace the old one with the new actually:
```suggestion
      exclude: ['*Vendor*', 'WordPress/WordPressTest/I18n.swift', 'WordPress/Classes/Utility/AppLocalizedString.swift'],
```

(I should have done that in https://github.com/wordpress-mobile/WordPress-iOS/pull/17630 alongside the file move but totally missed it back then)

File: fastlane/Fastfile (hunk 2 above)
Comment (AliSoftware) [reply fastlane/Fastfile:426]: That's correct and I should have removed those as part of those PRs, thanks for taking care of this.

There might still be an issue waiting to happen during `new_beta_release` and `finalize_release` when those will call the scripts downloading the translations from GlotPress and those scripts (which are still yet to be migrated) might still be trying to re-dispatch the keys to those widget targets based on those (now deleted) `Base.lproj` keys. Until that side of the migration of our scripts is over we might need to [patch the `extract-framework-translations.swift` scripts](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/Scripts/extract-framework-translations.swift#L10-L13) (or even stop calling that script altogether by removing [these lines](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/Scripts/update-translations.rb#L166-L168)) to stop doing that for those targets until I finalize the related work.

**Review by oguzkocer — APPROVED** [review]

**Pull request merged**